LICENSE.md

```diff
@@ -1,6 +1,4 @@
-## Released under MIT Licence
-
-The Emerald theme for Jekyll is Copyright (c) 2015 Jacopo Rabolini
+Copyright (c) 2018 CPSSD Students
```

Comment:
Disagree with this file rename. Regardless, there's no reason why this change should be part of this PR - put it in a different one.